build-system/global.d.ts

```diff
@@ -0,0 +1,8 @@
+declare global {
+  interface Error {
+    showStack?: boolean;
+    status?: string;
+  }
+}
```

Comment:
Can a comment be added to this file to explain that in the build-system, `Error` objects contain useful info like underlying stack traces in the `message` field, so we print that and hide the less useful nodejs stack.